wa/instruments/energy_measurement.py

```diff
@@ -153,6 +154,18 @@ def validate_parameters(self, params):
                 msg = 'Number of Energy Probe port labels does not match the number of resistor values.'
                 raise ConfigError(msg)
 
+class ArmEnergyProbeBackend(EnergyInstrumentBackend):
+
+    name = 'arm_energy_probe'
+
+    parameters = [
+        Parameter('config_file', kind=str, default='./config',
```

Comment:
Avoid relative paths in parameter values. 

The config files should by preserved as part of the output (probably in `__meta`) as it is essential for reproducing results.

Reply:
So I'm going to remove default path for config
ok. I'm going to copy the file in __meta

Reply:
How can I get path to __meta from EnergyInstrumentBackend ?
I have planned to copy the file from validate_parameters() but can't get the path where to copy the file

Reply:
The path can be obtained via `context.run_output.metadir` I would add that as an argument to `EnergyInstrumentBackend`'s  `__init__`, and it can then be passed inside `EnergyMeasurement.initialize`  via the `get_plugin` call. 

The actual copying can than happen inside your backend's `get_instruments` since this is also where it is used to initialize the `devlib` instrument.